parametric/protos/apm_test_client.proto

```diff
@@ -76,5 +88,76 @@ message FlushSpansReturn {}
 message FlushTraceStatsArgs {}
 message FlushTraceStatsReturn {}
 
+
+message OtelStartSpanArgs {
+  string name = 1;
+  optional bool new_root = 2;
+  optional uint64 parent_id = 3;
+  optional uint64 span_kind = 9;
+  optional string service = 4;
+  optional string resource = 5;
+  optional string type = 6;
+  optional int64 timestamp = 7;
+  optional attributes attributes = 8;
+}
+message attributes {
+  map<string, string> tags = 9;
+}
+
+message OtelStartSpanReturn {
+  uint64 span_id = 1;
+  uint64 trace_id = 2;
+}
+message OtelEndSpanArgs {
+  uint64 id = 1;
+}
+message OtelEndSpanReturn {}
+
+message OtelFlushSpansArgs {}
+message OtelFlushSpansReturn {}
+
+message OtelFlushTraceStatsArgs {}
+message OtelFlushTraceStatsReturn {}
+
+
+message OtelIsRecordingArgs {
+  uint64 span_id = 1;
+}
+message OtelIsRecordingReturn {
+  bool is_recording = 1;
+}
+
+message OtelSetStatusArgs {
+  uint64 span_id = 1;
+}
+message OtelSetStatusReturn {}
+
+message OtelSetNameArgs {
+  uint64 span_id = 1;
+  string name = 2;
+}
+message OtelSetNameReturn {}
+
+message OtelSetAttributesArgs {
+  uint64 span_id = 1;
+  map<string, string> attributes = 2;
+}
+message OtelSetAttributesReturn {}
+
+message OtelSpanContextArgs {
+  uint64 span_id = 1;
+}
+message OtelSpanContextReturn {
+  bytes span_id = 1;
+  bytes trace_id = 2;
+  int32 trace_flags = 3;
+  string trace_state = 4;
+  bool remote = 5;
+}
+
+
 message StopTracerArgs {}
 message StopTracerReturn {}
+
+message StartTracerArgs {}
```

Comment:
Should we expect future arguments?

Reply:
Yes, that would be configuration of the tracer/provider with environment / service or other Datadog supported tracer options